packages/react-composites/tests/browser/common/fixtureHelpers.ts

```diff
@@ -16,8 +16,20 @@ import { buildUrl } from './utils';
  * To be used in a playwright fixture's 'pages'.
  */
 // eslint-disable-next-line no-empty-pattern, @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/explicit-function-return-type
-export const usePagePerParticipant = async ({ serverUrl, users, browser }, use) => {
-  const pages = await Promise.all(users.map(async (user) => await loadNewPage(browser, buildUrl(serverUrl, user))));
+export const usePagePerParticipant = async (
+  {
+    serverUrl,
+    users,
+    browser
+  }: { serverUrl: string; users: Array<ChatUserType | CallUserType | CallWithChatUserType>; browser: Browser },
+  use: (pages: Page[]) => Promise<void>
```

Comment:
use `PlaywrightWorkerOptions` here instead (and for each instance in this PR)
```suggestion
  use: PlaywrightWorkerOptions
```